class.jetpack-cli.php

```diff
@@ -1476,6 +1476,156 @@ private function partner_provision_error( $error ) {
 		) ) );
 		exit( 1 );
 	}
+
+	/**
+	 * Creates the essential files in Jetpack to start building a Gutenberg block or plugin.
+	 *
+	 * ## TYPES
+	 *
+	 * block: it creates a Jetpack block. All files will be created in a directory under extensions/blocks named based on the block title or a specific given slug.
+	 *
+	 * ## BLOCK TYPE OPTIONS
+	 *
+	 * The first parameter is the block title and it's not associative. Add it wrapped in quotes.
```

Comment:
What does "it's not associative" mean here?

Reply:
In WPCLI, associative arguments are those like `--slug="the-block"`